Updated docs consider hosted ensnode instances (#413)
namehash/ensnode: docs/ensnode.io/src/content/docs/ensnode/usage/hosted-ensnode-instances.mdx

These instances are provided free of charge with no API key required, have no rate limiting, and are maintained and monitored by the NameHash Labs team.

:::caution[No Uptime / Performance Guarantees]
These hosted instances are provided free of charge to the ENS community, and while we strive for strong uptime and performance, neither of these can be guaranteed. By using these instances you accept the possibility of downtime, degraded performance, or other unforeseen events. If you'd like to run ENSNode in production, consider [hosting your own ENSNode instance](/ensnode/deploying/).
:::

## Available ENSNode Instances

> There are four ENSNode deployments which are serving indexed ENS data. All of them are separate deployments which are running on separate instances independently.

Reviewer: ```suggestion
NameHash Labs is hosting the following ENSNode deployments for the ENS community.
```

Author: Applied

> Main difference from client perspective is chain being indexed and list of plugins being handled. Here is the table listing differences between those.

Reviewer: ```suggestion
The key differences between these hosted deployments is the ENS deployment being indexed (mainnet vs a testnet) and the list of ENSNode plugins that are activated. See the table below for details.
```

Author: Applied

|                | **ENSNode 'Alpha'**                                |
| -------------- | --------------------------------------------       |
| URL            | [api.alpha.ensnode.io](https://api.alpha.ensnode.io) | 
| Active Plugins | eth, base.eth, linea.eth                           | 

> | Purpose        | Superset of ENS Subgraph names                     |

Reviewer: ```suggestion
| Purpose        | Demonstration of ENSNode's multichain ENS indexing capability. Provides a superset of the data indexed by the ENS Subgraph                     |
```
